app/views/devise/registrations/new.html.erb

```diff
@@ -3,6 +3,11 @@
     <h2 class=" offset-3 col-6"><%= t('.sign_up_page_title')%></h2>
     <%= simple_form_for(resource, as: resource_name, url: registration_path(resource_name), defaults: { input_html: { class: 'form-control' } }) do |f| %>
       <%= render "devise/shared/error_messages", resource: resource %>
+      <%#= f.input :first_name, placeholder: t(".first_name")%>
```

Comment:
why did you do it?

Reply:
I wanted to check if it was possible to register user with full data. but i was mistaken((

Reply:
I will remove it.